Sources/mpc-core-kit-swift/mpcCoreKitSwift.swift

```diff
@@ -223,8 +222,8 @@ public struct MpcCoreKit  {
         // to add tss pub details to corekit details
         let finalKeyDetails = try thresholdKey.get_key_details()
         let tssTag = try TssModule.get_tss_tag(threshold_key: thresholdKey)
-        let tssPubKey = try await TssModule.get_tss_pub_key(threshold_key: thresholdKey, tss_tag: tssTag)
-        return .init(tssPubKey: tssPubKey, metadataPubKey: try finalKeyDetails.pub_key.getPublicKey(format: .EllipticCompress), requiredFactors: finalKeyDetails.required_shares, threshold: finalKeyDetails.threshold, shareDescriptions: finalKeyDetails.share_descriptions, total_shares: finalKeyDetails.total_shares)
+        let tssPubKey = try? await TssModule.get_tss_pub_key(threshold_key: thresholdKey, tss_tag: tssTag)
```

Comment:
Shouldn't this always be available at this point? Why would we pass "" if it's null, how would we retrieve it later?

Reply:
It will be available if the tkey is reconstructed.